scripts/policy/tuning/__load__.zeek

```diff
@@ -1,2 +0,0 @@
-##! This loads the default tuning
-@load ./defaults
```

Comment:
I left this empty load script behind but it could easily be removed as well. We didn't deprecate loading the package itself, so removing the script would cause errors.